src/test/unit/services/scresults.spec.ts

```diff
@@ -0,0 +1,185 @@
+        callType: '0',
+        timestamp: 1678983282,
+        tokens: ['HYPE-619661'],
+        esdtValues: ['2568446286792506323583'],
+        operation: 'ESDTTransfer',
+      };
+
+      jest.spyOn(indexerService, 'getScResult').mockResolvedValue(mockIndexerScResult);
+      jest.spyOn(transactionActionService, 'getTransactionAction').mockResolvedValue({
+        category: 'esdtNft',
+        name: 'transfer',
+        description: 'Transfer',
+        arguments: {
+          transfers: [[Object]],
+          receiver: 'erd1ahauxqcaxvcksrcyyxzk5lrr3p8p8rjtayfue068nktrpjpvzers3dgvax',
+          functionName: undefined,
+          functionArgs: [],
+        },
+      });
+
+      const result = await service.getScResult(scHash);
+
+      expect(result).toBeInstanceOf(SmartContractResult);
+      expect(indexerService.getScResult).toHaveBeenCalledWith(scHash);
+      expect(indexerService.getScResult).toHaveBeenCalledTimes(1);
+      expect(transactionActionService.getTransactionAction).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe('getAccountScResults', () => {
+    const mockIndexerScResults = [{
+      scHash: 'f9a553996e7d3cde70d99212eb2462e209df721db324fdc72139626746369883',
+      nonce: 0,
+      gasLimit: '0',
+      gasPrice: '1000000000',
+      value: '0',
+      sender: 'erd1qqqqqqqqqqqqqpgql6dxenaameqn2uyyru3nmmpf7e95zmlxu7zskzpdcw',
+      receiver: 'erd1ahauxqcaxvcksrcyyxzk5lrr3p8p8rjtayfue068nktrpjpvzers3dgvax',
+      senderShard: 1,
+      receiverShard: 1,
+      data: 'RVNEVFRyYW5zZmVyQDQ4NTk1MDQ1MmQzNjMxMzkzNjM2MzFAOGIzYzVhNDFhMDRhNjliMjdm',
+      prevTxHash: 'f708864d802799353743f8703bffc87fa8167e46a522eb973fdfadedaa2bc9e0',
+      originalTxHash: 'f708864d802799353743f8703bffc87fa8167e46a522eb973fdfadedaa2bc9e0',
+      callType: '0',
+      timestamp: 1678983282,
+      tokens: ['HYPE-619661'],
+      esdtValues: ['1000000000000000000'],
+      operation: 'ESDTTransfer',
+    },
+    {
+      scHash: '6fc7c94e6fd113fd073e513d7e4f04d25bcbc4fefed8eea65748d18ed15b270d',
+      nonce: 0,
+      gasLimit: '0',
+      gasPrice: '1000000000',
+      value: '0',
+      sender: 'erd1qqqqqqqqqqqqqpgql6dxenaameqn2uyyru3nmmpf7e95zmlxu7zskzpdcw',
+      receiver: 'erd1ahauxqcaxvcksrcyyxzk5lrr3p8p8rjtayfue068nktrpjpvzers3dgvax',
+      senderShard: 1,
+      receiverShard: 1,
+      data: 'RVNEVFRyYW5zZmVyQDQ4NTk1MDQ1MmQzNjMxMzkzNjM2MzFAOGIzYzVhNDFhMDRhNjliMjdm',
+      prevTxHash: 'f708864d802799353743f8703bffc87fa8167e46a522eb973fdfadedaa2bc9e0',
+      originalTxHash: 'f708864d802799353743f8703bffc87fa8167e46a522eb973fdfadedaa2bc9e0',
+      callType: '0',
+      timestamp: 1678983282,
+      tokens: ['HYPE-619661'],
+      esdtValues: ['123416452926529462'],
+      operation: 'ESDTTransfer',
+    }];
+
+    const mockTransactionAction = {
+      category: 'esdtNft',
+      name: 'transfer',
+      description: 'Transfer',
+      arguments: {
+        transfers: [[Object]],
```

Comment:
Object should be changed with actual data

Reply:
fixed